spec/index.md

```diff
@@ -1308,6 +1308,7 @@ The following sections describe various System API functions, also referred to a
     ic0.global_timer_set : (timestamp : i64) -> i64;                            // I G U Ry Rt C T
     ic0.performance_counter : (counter_type : i32) -> (counter : i64);          // * s
     ic0.is_controller: (src: i32, size: i32) -> ( result: i32);                 // * s
+    ic0.in_replicated_execution: () -> (result: i32);                           // * 
```

Comment:
```suggestion
    ic0.is_replicated_execution: () -> (result: i32);                           // * 
```
just a suggestion

Reply:
Yeah -- I had is_in_replicated_execution to start with. The way I picked the current proposal is the canister asking "am I in replicated execution". 
Otherwise it should be "is_execution_replicated" :). 
let's not bikeshed too much -- of these options is one that you favor more?

Reply:
Makes sense. Let's stick with the current name.